features/step_definitions/basic_steps.rb

```diff
@@ -26,3 +26,7 @@
 When(/^I fill in "([^"]*)" with "([^"]*)"$/) do |element, content|
   fill_in element, with: content
 end
+
+When(/^I wait$/) do
```

Comment:
I would say 'I wait for the page to load' or embed this sleep in another step definition